bin/api_diff.ml

```diff
@@ -6,7 +8,13 @@ let run (`Ref_cmi reference) (`Current_cmi current) =
       ~current:current.cmi_sign
   in
   if differs = [] then Printf.printf "API unchanged!\n"
-  else Printf.printf "API changed!\n"
+  else
+    match differs with
+    | Api_watch_diff.Any :: _ -> Printf.printf "API changed!\n"
```

Comment:
I'd suggest dropping this header entirely and have the tool behave like `diff`:
1. exit with status 0 if there is no diff, no output
2. exit with status 1 if there is a diff, printing the diff

For `Any`, we already transform it into a textual diff so no need to have a specific treatment here.

Reply:
To deal with the exit status you can use `Cmdliner.Cmd.eval'` instead of the `Cmdliner.Cmd.eval` that we use and make our run function return the appropriate exit status instead of `()`.